### DISCUSSION TITLE
A proposal to take control of the Telegram `@craftcms` channel for community development.

### DISCUSSION THREAD

**Discussion by kksudo** [craftcms/cms#13709] [General]
## Description

I have the rights of the administrator of the [@craftcms](https://t.me/craftcms) Telegram channel, I want to suggest that you take control of this channel. I will give you administrator rights to manage this channel.

## Motivations:

Community development, answers to frequently asked questions, maintaining discussions in the messenger format.

P.S.: More than 7 years I switched from dev side to the DevOps side, and now I don't have enough time to develop my projects on the CraftCMS and be part of this community.

<details>
  <summary>Tg pic</summary>

![image](https://github.com/craftcms/cms/assets/8701324/2e216b0c-604b-4f2d-8e54-654b800af547)

</details>

**Comment by brandonkelly** [comment]
Hey @kksudo! Sure, we’d be happy to take that over. Can you reach out to us at support@craftcms.com?
